traffic_monitor/cache/cache.go

```diff
@@ -156,121 +145,142 @@ type Vitals struct {
 	MaxKbpsOut int64
 }
 
-// Stat is a generic stat, including the untyped value and the time the stat was taken.
+// Stat is a generic stat, including the untyped value and the time the stat was
+// taken.
 type Stat struct {
 	Time  int64       `json:"time"`
 	Value interface{} `json:"value"`
 }
 
-// Stats is designed for returning via the API. It contains result history for each cache, as well as common API data.
+// StatsCache is a representation of cache server statistics as present in the
+// TM API.
+type StatsCache struct {
```

Comment:
This is a bit of a nitpick, but this naming is _very_ confusing to me.
You have a Stats object that has a map of StatsCache that has a map of statistics called Stats.
Perhaps the best solution would be to rename the Stats struct?

Reply:
Sure - to what?

Reply:
`ServerStats`?